ksql-engine/src/main/java/io/confluent/ksql/services/KafkaTopicClientImpl.java

```diff
@@ -261,7 +263,7 @@ public void deleteTopics(final Collection<String> topicsToDelete) {
 
   @Override
   public void deleteInternalTopics(final String applicationId) {
-    if (!isDeleteTopicEnabled) {
+    if (!isTopicDeleteEnabled()) {
```

Comment:
this will be called both here and later in the method (line 279) when we call deleteTopics. we should cache the result and pass it down (alternatively, we can wrap it in a MemoizedSupplier if this client is created once per user/request)

Reply:
Thanks. I used the MemoizedSupplier